developer_tests/location/run_tests.csh

```diff
@@ -3,80 +3,84 @@
 # DART software - Copyright UCAR. This open source software is provided
 # by UCAR, "as is", without charge, subject to all terms of use at
 # http://www.image.ucar.edu/DAReS/DART/DART_download
-
+#
 # this script builds and  runs the location test code for each of the
 # possible location modules.
+#
+#----------------------------------------------------------------------
 
-set nonomatch
-set LOGDIR = `pwd`/testing_logs
-mkdir -p $LOGDIR
-\rm -f $LOGDIR/*
-echo "build and run logs are in: $LOGDIR"
+# prevent shell warning messages about no files found when trying
+# to remove files using wildcards.
 
+set nonomatch
 
 echo
 echo
 echo "=================================================================="
-echo "Starting location module tests at "`date`
+echo "Start of location module tests at "`date`
 echo "=================================================================="
 echo
 echo
 
+set LOGDIR = `pwd`/testing_logs
+mkdir -p $LOGDIR
+\rm -f $LOGDIR/*
+echo "build and run logs are in: $LOGDIR"
 
 set LOCLIST = ( annulus channel column oned threed \
                 threed_cartesian threed_sphere \
                 twod twod_annulus twod_sphere )
 
 foreach i ( $LOCLIST )
 
- echo
- echo
- echo "=================================================================="
- echo "Starting tests of location module $i at "`date`
- echo "=================================================================="
- echo
- echo
-
- set FAILURE = 0
-
- cd $i/test
-
- ./mkmf_location_test
- ( make > $LOGDIR/buildlog.$i.out ) || set FAILURE = 1
-
- echo
- echo
- if ( $FAILURE ) then
-   echo "=================================================================="
-   echo "ERROR - unsuccessful build of location module $i at "`date`
-   echo "=================================================================="
    echo
    echo
- else
+   echo "------------------------------------------------------------------"
+   echo "Starting tests of location module $i at "`date`
+   echo "------------------------------------------------------------------"
+   echo
+   echo
+
+   set FAILURE = 0
```

Comment:
Sorry about this ... these are whitespace changes. They were indented improperly originally (1 character, which is not much of an indent at all!)